features/polyExplorer/src/screens/entityDetails/entityDetails.css

```diff
@@ -307,14 +310,8 @@
 }
 
 .about .entity-details-text {
-    overflow: hidden scroll;
-    max-height: 300px;
-    margin: 32px 16px 0 16px;
-}
-.about .scroll-box {
-    overflow: hidden scroll;
     max-height: 300px;
```

Comment:
Since we don't need the scrolling anymore, we also need to delete this `max-height`, otherwise the text will mixed in the background of the content below in those cases where the text is very long.